hwdb.d/60-keyboard.hwdb

```diff
@@ -91,6 +92,17 @@
 # Examples of such devices: Chromebooks where the top row is used for both
 # media and F1-F10 keys.
 
+# ######################### WAKEUP KEYS ######################################
+#
+# The wakeup keys specify which keys unblank the screen of an idle device. By
+# default all keys are wakeup keys The default can be changed on a per keyboard
+# level via
+#  WAKEUP_KEY_DEFAULT=0
```

Comment:
Not super-happy with this name, `WAKEUP_DEFAULT=0` is a bit meaningless, it doesn't explain if that's a state, a default key, 0-to-disable, etc.

Incomplete ideas:
- `WAKEUP_KEYS=all|none`
- `WAKEUP_KEYS_ALLOW_WAKEUP=1`

Struggling to come up with something better...

But going with my comment above, there seems to be a use-case for differentiating between waking up userspace, the screen, clients, etc. and a 1 or 0 won't be enough for that.